advanced_functionality/pipe_bring_your_own/pipe_bring_your_own.ipynb

```diff
@@ -203,6 +203,7 @@
     "%%sh\n",
     "REGION=$(aws configure get region)\n",
     "account=$(aws sts get-caller-identity --query Account --output text)\n",
+    "docker login --username AWS --password $(aws ecr get-login-password --region us-west-2) 763104351884.dkr.ecr.us-west-2.amazonaws.com\n",
```

Comment:
Can you keep the use of `${REGION}` rather than hard-coding us-west-2?

Reply:
Not for this login - this login is specifically for pulling DLC images from the DLC account. I would ideally, however the account number changes as the region change. I decide to set it to download from us-west-2 for now, I may have to find another solution for internet-free in the future (in fact, whether docker can be internet-free is a question mark in itself as we sometimes use external base images)

Reply:
Okay - I think there are edge cases where this could break, but understood that it's pulling a particular account ID which can vary across regions.